src/senseglove/senseglove_finger_distance/scripts/senseglove_finger_distance_node

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python2.7
```

Comment:
does this mean 2.7 is always used? is the support for python 2 due to use of ROS1?

Reply:
2.7 is used because this is the most recent Python version that Melodic supports. Whenever we will upgrade to Noetic this will switch to Python 3.